docs/content/preview/yugabyte-voyager/release-notes.md

```diff
@@ -23,6 +23,26 @@ What follows are the release notes for the YugabyteDB Voyager v1 release series.
 
 Voyager releases (starting with v2025.5.2) use the numbering format `YYYY.M.N`, where `YYYY` is the release year, `M` is the month, and `N` is the number of the release in that month.
 
+## v2025.9.1 - September 2, 2025
+
+### New feature
+
+- Export schema automatically exports all secondary indexes as range-sharded by default to avoid potential hash hotspots. Use the [skip-performance-recommendations](../reference/schema-migration/export-schema/#arguments) flag to skip this automatic change.
+
+### Enhancements
+
+- The assessment report now includes primary key recommendations for tables having no primary key but with UNIQUE NOT NULL columns, improving schema optimization guidance.
+
+- Improved the estimated time for import calculations in the assessment report by additionally estimating the time without considering redundant indexes, which are now automatically excluded in the export schema step.
+
+- End migration now backs up the schema optimization report produced by export schema.
+
+- Modified flag names in export schema from "skip-performance-optimizations" to "skip-performance-optimizations" and from "skip-recommendations" to "skip-colocation-recommendations" for consistency and clarity.
+
+### Bug fix
+
+- Fixed an issue where source password wasn't being propagated to assess-migration when run internally in export schema if not set via the CLI flag.
```

Comment:
hard to follow. explain.

Reply:
More context: 
When running export-schema, if assess-migration was not already run before, it is run automatically. 
The bug: 
When running export-schema, if you do not specify the source db password via CLI (you specify it via an ENV var or as console input when prompted by voyager), then the password was not correctly configured when automatically running assess-migration.